apps/web/types.tsx

```
@@ -11,6 +11,9 @@ export type ScreenContext = {
   req: GetServerSidePropsContext['req']
 }
 
-export type Screen<Props = {}> = ComponentType<Props> & {
-  getProps: (ctx: ScreenContext) => Promise<Props>
+export type Screen<
+  Props = {},
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)